ui-react/packages/atlasmap-ui/src/CanvasView/components/CanvasViewToolbar.tsx

```diff
@@ -35,18 +45,23 @@ export const CanvasViewToolbar: FunctionComponent<ICanvasViewToolbarProps> = ({
           <Button
             variant={'plain'}
             aria-label="Enable/ Disable conditional mapping expression"
-            onClick={onToggleExpressionMode}
+            onClick={() => {
+              onToggleExpressionMode();
+              // setCondExprEnabled(onConditionalMappingExpressionEnabled());
+              condExprEnabled = onConditionalMappingExpressionEnabled();
```

Comment:
Sorry @pleacu, but this is the same thing as the previous comment. You can't change a local variable in an event handler in React, it makes no sense.

Reply:
Yes - will address that today.  My issue was that I was calling onConditionalMappingExpressionEnabled() to set a useState boolean.  The core service function (conditionalMappingExpressionEnabled) returns true but the onConditionalMappingExpressionEnabled() returns false in the onClick():

setCondExprEnabled(onConditionalMappingExpressionEnabled());

Is there a special syntax to invoke onConditionalMappingExpressionEnabled()??